superset-frontend/src/components/Chart/ChartErrorMessage.tsx

```diff
@@ -21,10 +21,16 @@ import React from 'react';
 import { SupersetError } from '@superset-ui/core';
 import { useChartOwnerNames } from 'src/hooks/apiResources';
 import ErrorMessageWithStackTrace from 'src/components/ErrorMessage/ErrorMessageWithStackTrace';
+import { ChartSource } from 'src/types/ChartSource';
 
-interface Props {
+export interface Props {
   chartId: string;
   error?: SupersetError;
+  subtitle: JSX.Element;
+  copyText: string;
+  link?: string;
+  source: ChartSource;
+  stackTrace?: string;
```

Comment:
I believe this could be an extension of `superset-frontend/packages/superset-ui-core/src/query/getClientErrorObject.ts` `ClientErrorObject`